src/rest/Makefile

```diff
@@ -22,7 +22,7 @@ ABS_TOPDIR ?= $(abspath $(TOPDIR))
 BUILD_DIR  ?= $(TOPDIR)/build
 
 GO     ?= go
-GOPATH ?= $(ABS_TOPDIR)/gopkgs:$(ABS_TOPDIR):$(shell $(GO) env GOPATH)
+GOPATH ?= $(BUILD_GOPATH):$(ABS_TOPDIR):$(shell $(GO) env GOPATH)
```

Comment:
BUILD_GOPATH is not initialized locally and hence this makefile cannot be run on its own now. Note every variable is set only if it was not set already (from top makefile). This way it can be run from top makefile or on its own. We need this for incremental build of rest server - for quick verification of REST server changes..

Reply:
"make rest-server" from top level will work.
I will add it anyway though.